.rubocop.yml

```diff
@@ -1158,4 +1158,4 @@ Style/WordArray:
   Enabled: false
 
 Style/FrozenStringLiteralComment:
-  Enabled: true
+  Enabled: false
```

Comment:
The way to fix it is actually freezing those strings, not disabling the linter unless it's not wanted anymore...